src/services/userService.ts

```diff
@@ -96,39 +112,46 @@ async function updateUserInfo(userId, updatedData) {
   return filterSensitiveUserData(updatedUser);
 }
 
-async function changePassword(userId, currentPassword, newPassword) {
+async function changePassword(
+  userId: string,
+  currentPassword: string,
+  newPassword: string
+) {
   const user = await userRepository.findById(userId);
   if (!user) {
-    const error = new Error("User not found");
+    const error: any = new Error("User not found");
```

Comment:
Error 객체를 서언 했는데, 여기서 다시 변수의 타입을 any로 정의하신 이유가 있으실까요?
```suggestion
    const error = new Error("User not found");
```